collect_app/src/androidTest/java/org/odk/collect/android/feature/projects/UpdateProjectTest.kt

```diff
@@ -0,0 +1,39 @@
+package org.odk.collect.android.feature.projects
+
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.pressBack
+import androidx.test.espresso.matcher.ViewMatchers
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.RuleChain
+import org.odk.collect.android.support.CollectTestRule
+import org.odk.collect.android.support.TestRuleChain
+
+class UpdateProjectTest {
+
+    val rule = CollectTestRule()
+
+    @get:Rule
+    var chain: RuleChain = TestRuleChain
+        .chain()
+        .around(rule)
+
+    @Test
+    fun updateProjectTest() {
+        rule.mainMenu()
+            .assertProjectIcon("D", "#3e9fcc")
+            .openProjectSettingsDialog()
+            .assertCurrentProject("Demo project")
+            .clickAdminSettings()
+            .setProjectName("Project X")
+            .setProjectIcon("X")
+            .setProjectColor("#cccccc")
+
+        onView(ViewMatchers.isRoot()).perform(pressBack())
```

Comment:
I could use `Espresso.pressBack()` we also have the` pressBack()` method in our Page class but it doesn't work. I ran into the same problem again and spent some time investigating it turns out it doesn't work only in kotlin tests... strange but I don't know what the cause is.

Reply:
Oh wow that sucks. Do you want to just change the implementation of `pressBack` on `Page` then so we can still chain with it?

Reply:
I tried it but the same line that works here doesn't work if it's in java class (and called here from kotlin). Probably I would need to convert those page classes to kotlin but that's what I don't want to do as a part of this or.

There is something strange between kotlin and java here.